lesson_4/customerForm.js

```diff
@@ -0,0 +1,52 @@
+document.addEventListener('DOMContentLoaded', () => {
```

Comment:
As far as we don't use our DOM in the closest lessons you could avoid this `eventListener`

Reply:
ok